Assets/Scripts/ThirdPersonCamera.cs

```diff
@@ -16,15 +16,25 @@ public class ThirdPersonCamera : MonoBehaviour
 
     private Transform _cameraTarget;
     private Vector3 _currentCameraVelocity = Vector3.zero;
- 
-    private bool _levelCleared;
-    public Transform topViewCamPos;
+
 
     //InputActions
     PlayerInputActions inputAction;
 
     Vector2 cameraRotationInput;
 
+    private bool _enableIsometricView;
+    private bool _levelCleared;
+
+    private Transform _viewPos;
+    public Transform constellationViewPos;
+    public Transform isometricStarViewPos;
+    
+    private bool _returnToPlayer;
+    private bool _rotateToTopView;
+    private bool _onOrbit;
+    private bool _onIso;
```

Comment:
How is _onOrbit to the camera when it's only affected by whether or not it is _onIso?